pytests/tests/test_cache.py

```diff
@@ -218,15 +214,18 @@ def test_download_vs_cache_size_multiple_packages(utils):
     assert utils.floats_approx_equal(down_bytes, cached_rpm_bytes)
 
 
-@pytest.mark.skipif(try_mount_small_cache() != 0, reason="Failed to mount small cache directory.")
 def test_cache_directory_out_of_disk_space(utils):
+    if try_mount_small_cache():
+        print("Failed to mount small cache directory")
+        return
+
     small_cache_path = utils.config['small_cache_path']
     orig_cache_path = utils.tdnf_config.get('main', 'cachedir')
     switch_cache_path(utils, small_cache_path)
     enable_cache(utils)
-    clean_small_cache(utils)
+    utils.run(f"rm -rf {small_cache_path}/*".split())
```

Comment:
I added a helper function for this, `shutil.rmtree` deletes the directory, not just the content within. Here `small_cache_path` is a mount point, so it will fail.